static/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -9,17 +9,26 @@ Once initialized in a project, DVC populates its installation directory
 - `.dvc/config.local` - this is a local configuration file, that will overwrite
   options in `.dvc/config`. This is useful when you need to specify private
   options in your config, that you don't want to track and share through git.
-  The local config file can be edited by hand or with a special command: `dvc
-  config --local`.
-
-- `.dvc/cache` - the cache directory will contain your data files. (The data
+  The local config file can be edited by hand or with a special command:
+  `dvc config --local`.
+
+- `.dvc/cache` - the cache directory will contain your data files. The data file
+  is converted to a MD5 checksum which is a 32 characters long string. The first
+  two characters are assigned to name the directory inside `.dvc/cache` and rest
+  are given to name the cache file. For example, if a data file, say
+  `Posts.xml.zip`, is converted to a MD5 checksum, it will evaluate to
+  `ec1d2935f811b77cc49b031b999cbf17`. The cache file for this data file will be
+  stored as `.dvc/ec/1d2935f811b77cc49b031b999cbf17` on the local storage and if
+  it is pushed to a remote storage, its location will be
+  `<prefix>/ec/1d2935f811b77cc49b031b999cbf17` where prefix is the name of the
+  remote storage. `/tmp/dvc-storage` can be one example of a prefix. (The data
```

Comment:
we need to specify how we deal with directories

Reply:
@shcheklein I can't follow. Could you please elaborate? :sweat_smile: